plugin/src/main/java/com/denizenscript/denizen/objects/EntityTag.java

```diff
@@ -2887,6 +2888,23 @@ else if (object.getBukkitEntity() instanceof Hanging) {
             return new ElementTag(humanEntity.isHandRaised());
         });
 
+        // <--[tag]
+        // @attribute <EntityTag.bookshelf_slot>
+        // @returns ElementTag(Number)
+        // @description
+        // Returns the Chiseled Bookshelf slot that the entity is looking at.

```

Comment:
`, if any`
Also, should link the actual slot tag for more advanced usage - just a `See also <@link>.` should be fine.